examples/blend/main.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2023 The Ebitengine Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"image"
+	"image/color"
+	_ "image/png"
+	"log"
+
+	"golang.org/x/image/font/inconsolata"
+
+	"github.com/hajimehoshi/ebiten/v2"
+	"github.com/hajimehoshi/ebiten/v2/examples/resources/images/blend"
+	"github.com/hajimehoshi/ebiten/v2/text"
+)
+
+const (
+	screenWidth  = 780
+	screenHeight = 600
+	tileSpacing  = 64
+	textSpacing  = 16
+	gridW        = 4
+	gridH        = 3
```

Comment:
```go
	tileSpacing  = 64
	textSpacing  = 16
	gridW        = 4
	gridH        = 3
```

I don't think we need these consts for Game's member variables.

I suggested to remove Game's members and replace them with the consts, but this is optional.

Reply:
I don't know, the Game will have a tighter coupling then, no?

Reply:
So you mean you prefer keeping member variables? Then remove these consts.

Reply:
Idk, the consts are more convenient. But can't we kept both? Then the user can just grab the Game implementation and use it in he's own code.

Reply:
I don't think so since they are redundant. If we keep constants, 1) move the four constsnts to Draw and 2) remove the member variables. 

> Then the user will be able to just grab and use the Game implementation in his code.

In terms of reusability, I think consts in Draw is enough.

Reply:
Removed constructor args

Reply:
Oh, I didn't see that...
Shove them right inside the draw? Do you mean like this?
```go
func (g *Game) Draw(screen *ebiten.Image) {
	screen.Fill(color.White)
	const (
		tileSpacing = 64
		textSpacing = 16
		gridW       = 4
		gridH       = 3
	)
	...
}	
```
Hm, it looks strange.
Is it makes sense to separate them, why just not let user edit them in order to extend the grid?

Reply:
> Do you mean like this?

Yes

> Is it makes sense to separate them,

I think this makes sense since the smaller scoper are the better they are in general.

> why just not let user edit them in order to extend the grid?

I don't understand. The users still can edit the consts whererever they are.

Reply:
Fixed